src/game/core/entity/Entity.ts

```diff
@@ -0,0 +1,30 @@
+import { TCoordinates, TSize } from 'game/core/types';
+import { Stage, Sprite } from 'game/core';
+
+export default class Entity {
+  private _sprite: Sprite;
+  position: TCoordinates;
+  speed: number | null
+
+  constructor(initialPosition: TCoordinates, sprite: Sprite, speed?: number) {
+    this.position = initialPosition;
+    this._sprite = sprite;
+    this.speed = speed || null;
```

Comment:
❗️
Лучше группировать "приватные" и публичные поля
```suggestion
    this._sprite = sprite;
    
    this.position = initialPosition;
    this.speed = speed || null;
```